src/styles/watch.scss

```diff
@@ -0,0 +1,63 @@
+.watch {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  height: $watchSize;
+  width: $watchSize;
+  
+  margin: 0 auto;
+  padding: 5px;
+
+  border: 4px solid #062550;
+  border-radius: 50%;
+  background-color: #c396aa;
+
+  &__minute-arrow {
+    position: absolute;
+    height: $arrowHeight;
+    width: $arrowWidth;
+
+    background-color: #451a49;
+    border-radius: 8px;
+
+    transform-origin: bottom;
+    animation-name: move;
+    animation-duration: 3660s;
+    animation-timing-function: linear;
+  }
+
+  &__second-arrow {
+    position: absolute;
+    height: $arrowHeight;
+    width: $arrowWidth;
+
+    background-color: #062550;
+    border-radius: 8px;
+
+    transform-origin: bottom;
+    animation-name: move;
+    animation-duration: 60s;
+    animation-iteration-count: infinite;
+    animation-timing-function: steps(60);
```

Comment:
Here too.
```suggestion
    animation-name: move 60s infinite steps(60);
```

Reply:
Thank you for review! 
I've wrote it like that on purpose, for my understanding how it works in general, in next tasks i will use the short code)))

Reply:
> Thank you for review! I've wrote it like that on purpose, for my understanding how it works in general, in next tasks i will use the short code)))

oh, now I got it )